applications/MeshingApplication/python_scripts/mmg_process.py

```diff
@@ -155,6 +155,8 @@ def __init__(self, Model, settings ):
             "save_external_files"              : false,
             "save_colors_files"                : false,
             "save_mdpa_file"                   : false,
+            "remesh_post_process"              : true,
```

Comment:
I think we need a better name here:

```suggestion
            "remesh_at_finalize"              : true,
```

Also we lack another boolean to control the output of the mesh!

Reply:
I changed the name, and added the boolean

Reply:
I think you still need to remove the setting `"remesh_post_process"`, as it is not used in the code